cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
remove this folder, should not be used for now (it was automatically installed by cypress)